### PR TITLE
Add support for redirectUrl on /create route

### PR DESCRIPTION
**Pull request by Patrick1904**
**Changes:**
1. Add support for optional `redirectUrl`
2. Add `accountId` as url param to `redirectUrl`

**Example:**
`https://wallet.near.org/create?redirectUrl=https://berryclub.io/`

After account creation --> `https://berryclub.io/?accountId=johndoe.near`

**W/ linkdrop example:**
`https://wallet.near.org/create/near/VGmU9e2cEgE8PHysrBdjeJkbzvvFtBYT1UgEmDHTFTp81gnHSdKZCVyVpV5PwjYT6FX7Znd731HCvSpVnJJmgnx?redirectUrl=https://berryclub.io/`

After account creation --> `https://berryclub.io/?accountId=johndoe.near`

### CODE REVIEW

**Comment by vgrichina** [comment]
nit: I'd prefer we call it `redirectUrl` as described here https://github.com/near/near-wallet/issues/1641 to avoid confusion with login `success_url` which is supposed to be deprecated when we implement https://github.com/near/near-wallet/issues/726

File: src/actions/account.js [hunk 1]
```diff
@@ -73,24 +73,23 @@ export const parseTransactionsToSign = createAction('PARSE_TRANSACTIONS_TO_SIGN'
...
-
     if ([...WALLET_CREATE_NEW_ACCOUNT_FLOW_URLS, WALLET_LOGIN_URL, WALLET_SIGN_URL].includes(currentPage)) {
         const parsedUrl = {
             referrer: document.referrer && new URL(document.referrer).hostname,
             ...parse(search),
             redirect_url: prevRouter ? prevRouter.location.pathname : undefined
         }
-
-        if ([WALLET_LOGIN_URL, WALLET_SIGN_URL].includes(currentPage) && search !== '') {
+        if (['create'].includes(currentPage) && search !== '') {
```
Comment (vgrichina) [inline src/actions/account.js:82]: nit: let's use constant for 'create' like for other paths

File: src/actions/account.js [hunk 2]
```diff
@@ -414,18 +413,22 @@ export const handleCreateAccountWithSeedPhrase = (accountId, recoveryKeyPair, fu
...
-    if (new BN(account.balance.available).lt(new BN(utils.format.parseNearAmount(MULTISIG_MIN_PROMPT_AMOUNT)))) {
+    if (new BN(balance.available).lt(new BN(utils.format.parseNearAmount(MULTISIG_MIN_PROMPT_AMOUNT)))) {
         promptTwoFactor = false
     }
 
     if (promptTwoFactor) {
         dispatch(redirectTo('/enable-two-factor', { globalAlertPreventClear: true }))
     } else {
-        dispatch(redirectToApp('/profile'))
+        if (url?.redirect_url.includes('/create') && url?.success_url) {
```
Comment (vgrichina) [inline src/actions/account.js:427]: not sure how `redirect_url` comes into play here?

**Comment by Patrick1904** [comment]
> nit: I'd prefer we call it `redirectUrl` as described here #1641 to avoid confusion with login `success_url` which is supposed to be deprecated when we implement #726

Thanks @vgrichina. I simplified it a bit and fixed the naming. Please check it.

**Pull request merged**